end-to-end-tests/src/bin/bootstrap.rs

```diff
@@ -1,11 +1,36 @@
+// TODO - levon
+// cleanup
```

Comment:
Does this refer to the dead-code in this file? Might be good to keep a patch locally or in a WIP branch, but not merged to `main`?

Reply:
Yep, dead code, I've already removed it locally, going to push with the latest rebase